src/components/ThemeToggle/ThemeToggle.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useTheme } from 'next-themes'
+import styles from './ThemeToggle.module.scss'
+
+const ThemeToggle = () => {
+  const { theme, setTheme } = useTheme()
+
+  return (
+    <button
+      type="button"
+      onClick={() => setTheme(theme === 'light' ? 'dark' : 'light')}
+      className={styles.toggleButton}
+      data-testid="theme-toggle">
+      {theme === 'light' ? 'dark' : 'light'} mode
```

Comment:
Same here

```suggestion
      {theme === 'dark' ? 'dark' : 'light'} mode
```

Reply:
Looking at the design in Figma, it's showing that while in dark mode, the button says 'Light Mode'. I just assumed that the button text was meant to indicate what theme you would like to change too. Maybe we should revisit this when Mallory is back? I tried to find an example of the inverse, but haven't been able to.

Reply:
^^ I remember seeing this in figma and thinking it was really confusing.

Reply:
This is confusing to me as well but now that you explained it I can see how they intend it. Maybe eventually it will become a sliding switch like what some other sites have. Another good thing to add to the list for when Mallory is back. Thanks for showing my mis-understanding! ❤️